src/com/collection/tree/MyTreeMap.java

```diff
@@ -0,0 +1,32 @@
+package com.collection.tree;
+
+import java.util.Set;
+import java.util.TreeSet;
+
+public class MyTreeMap<V extends Comparable<V>> implements Tree<V> {
+    private final Set<V> values = new TreeSet<>();
```

Comment:
Lets try to write your own implementation of the tree instead of using existing one.